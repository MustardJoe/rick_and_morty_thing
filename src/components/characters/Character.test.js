import React from 'react';
import { shallow } from 'enzyme';
import Character from './Character';

describe('Character comp test', () => {
  it('renders a character', () => {
    const character = {
      name: 'morthy',
      status: 'dead',
      species: 'klingon',
      image: '/image'
    };

    const wrapper = shallow(<Character character={character} />);
    expect(wrapper).toMatchSnapshot();
  });
});
