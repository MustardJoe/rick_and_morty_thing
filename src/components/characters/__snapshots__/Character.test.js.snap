// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character comp test renders a character 1`] = `
<section>
  <img
    src="/image"
  />
  <p>
    morthy
  </p>
  <p>
    dead
  </p>
  <p>
    klingon
  </p>
</section>
`;
